RecoEgamma/EgammaTools/plugins/CandiateCombiner.cc

```diff
@@ -0,0 +1,120 @@
+//****************************************************************
+//
+// A simple class to combine to candidate collections into a single
```

Comment:
```to``` -> ```two ``` ?

Reply:
There is nothing egamma-specific here.
I think that this file can be moved to CommonTools/CandAlgos and renamed to CandMergerCleanOthersByDr.

Reply:
I was also thinking that might be a good idea but was at a loss where to stick it. Agreed and done.